Makefile

```
@@ -11,7 +11,17 @@ lint::
 	golangci-lint run --timeout 5m
 
 test_acceptance::
-	go test -v -count=1 -cover -timeout 2h -parallel ${TESTPARALLELISM} ./tests/...
+	go test -v -count=1 -cover -timeout 2h -parallel ${TESTPARALLELISM} ./tests/terraform/...
+
+generate_tf2pulumi_coverage_input::
+	(cd tests/coverage-report/testdata && if [ ! -d terraform-provider-aws ]; then git clone https://github.com/hashicorp/terraform-provider-aws && cd terraform-provider-aws && git checkout 59d66d6283496aa47e90ec78d0eb3851e0a640e1; fi)
+	(cd tests/coverage-report/testdata && if [ ! -d terraform-provider-azurerm ]; then git clone https://github.com/terraform-providers/terraform-provider-azurerm && cd terraform-provider-azurerm && git checkout 8fc7613206855de21b1771a209510890f701a24b; fi)
+	(cd tests/coverage-report/testdata && if [ ! -d terraform-provider-google ]; then git clone https://github.com/hashicorp/terraform-provider-google && cd terraform-provider-google && git checkout ce331bb9c3984301ac3e7132a49aebb93e656832; fi)
+	(cd tests/coverage-report/testdata && if [ ! -d example-snippets ]; then cd ../test && go generate; fi)
+
+tf2pulumi_coverage_report::
+	make generate_tf2pulumi_coverage_input
```

You can append any make targets that need to run as part of this target.
```suggestion
tf2pulumi_coverage_report:: generate_tf2pulumi_coverage_input
```